config/routes.rb

```diff
@@ -170,12 +224,10 @@
       end
     end
 
-    unless Rails.env.production?
-      get 'raise_exception/:type', to: 'dev#raise_exception'
-    end
+    get 'raise_exception/:type', to: 'dev#raise_exception' unless Rails.env.production?
   end
 
-  use_doorkeeper { controllers applications: 'oauth/applications' }
+  use_doorkeeper do controllers applications: 'oauth/applications' end
```

Comment:
Single line do...end is a bit weird to me.

Reply:
Fixed.